LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/IT87XX.cs

```diff
@@ -643,7 +644,7 @@ private void RestoreDefaultFanPwmControl(int index)
         private const byte VOLTAGE_BASE_REG = 0x20;
 
         private readonly byte[] FAN_PWM_CTRL_REG;
-        private readonly byte[] FAN_PWM_CTRL_EXT_REG = { 0x63, 0x6b, 0x73, 0x7b, 0xa3 };
+        private readonly byte[] FAN_PWM_CTRL_EXT_REG = { 0x63, 0x6b, 0x73, 0x7b, 0xa3, 0xab};
```

Comment:
This number was arrived by guesswork, however I am able to control and monitor all 6 fan headers in practice.

Reply:
This number is also used by lm-sensors so should be OK.